astropy/io/fits/hdu/table.py

```diff
@@ -790,6 +785,30 @@ class BinTableHDU(_TableBaseHDU):
     _extension = 'BINTABLE'
     _ext_comment = 'binary table extension'
 
+    def __init__(self, data=None, header=None, name=None, uint=False):
+        """
+        Parameters
+        ----------
+        header : `Header`
+            header to be used
+        data : array, `FITS_rec` or `astropy.table.Table`
```

Comment:
There is no `Table` class in `io.fits` but you can leave it as-is (without tilda) if you want.